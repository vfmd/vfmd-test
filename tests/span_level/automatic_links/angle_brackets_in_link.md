- http://exampl<e.ne>t/ and <http://exampl<e.ne>t/>
- someone@exampl<e.ne>t and <someone@exampl<e.ne>t>
