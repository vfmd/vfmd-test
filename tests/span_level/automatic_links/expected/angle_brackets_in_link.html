<ul>
  <li><a href="http://exampl">http://exampl</a>&lt;e.ne&gt;t/ and &lt;<a href="http://exampl">http://exampl</a>&lt;e.ne&gt;t/&gt;</li>
  <li>someone@exampl&lt;e.ne&gt;t and &lt;someone@exampl&lt;e.ne&gt;t&gt;</li>
</ul>
